vochain/state/utils.go

```diff
@@ -31,6 +31,18 @@ func GetFriendlyResults(results []*models.QuestionResult) [][]string {
 	return r
 }
 
+// ResultsToBigIntMatrix converts the results of a process to a big.Int matrix.
+func ResultsToBigIntMatrix(results []*models.QuestionResult) [][]*big.Int {
```

Comment:
We are repeating code. What about modifying GetFriendlyResults to return a `[][]*types.bigInt` instead of [][]string ?

When types.bigInt is stringified or json-marshaled, it correctly converts the contented to a String.

Reply:
Idk, then we are going to repeat code in other parts of the code. 2 times 2 nested loops.

Reply:
Made the proper changes for enabling this changes in the last commit :)